llmos-ai/llmos-operator
# Move llmos dependency charts to system-addons

**IMPORTANT: Please do not create a Pull Request without creating an issue first.**

**Problem:**
Moving `llmos-operator` dependency charts to system-addon for better extensibility and management.

**Test:**
1. install the llmos-operator chart with the latest version.
2. check if managedaddon CRDs are created and its related helm charts are installed correctly.
3. check if pods are running correctly.
